assets/css/calypsoify.css

```diff
@@ -558,10 +558,38 @@ table.wc-shipping-classes td.wc-shipping-zone-method-blank-state, table.wc-shipp
 .wc-wizard-service-item .wc-wizard-service-name, .wc-wizard-services-list-toggle .wc-wizard-service-name {
 	min-width: 100px;
 	padding: 20px;
+	display: flex;
+    align-items: center;
 }
 .wc-wizard-service-item .wc-wizard-service-enable, .wc-wizard-services-list-toggle .wc-wizard-service-enable {
 	align-self: center;
 }
+.wc-wizard-service-item.square-logo .wc-wizard-service-name, .wc-wizard-services-list-toggle.square-logo .wc-wizard-service-name,
+.wc-wizard-service-item.klarna-logo .wc-wizard-service-name, .wc-wizard-services-list-toggle.klarna-logo .wc-wizard-service-name {
+    background: #fff;
+}
+.wc-wizard-service-item.square-logo .wc-wizard-service-name:before,
+.wc-wizard-service-item.klarna-logo .wc-wizard-service-name:before {
+    content: '';
+    display: block;
+    margin: 0 auto;
+	padding: 2px 0;
```

Comment:
It looks like there is some indentation issues here.

Reply:
Fixed- thanks!